source/Ship.cpp

```diff
@@ -3554,7 +3554,7 @@ double Ship::MaxVelocity() const
 
 double Ship::ReverseAcceleration() const
 {
-	return attributes.Get("reverse thrust");
+	return attributes.Get("reverse thrust") / InertialMass();
```

Comment:
The existing version is indeed incorrect. Can you pull this change out (or, do you mind if I do) into another PR?
I don't expect to get to the rest of this before the next release but I will merge this part.

Reply:
Please go ahead, if you will safe time by doing so (otherwise please just tell me and I'll do it).

That being said, I split this PR from the rest on purpose, as this PR can't do harm in my opinion (famous last words). The code path for all "regular" ships is basically unchanged (only indented).
It's only the player's reverse thruster heavy ships (as there are no NPCs with that as far as I am aware) which are changed and their new behavior is a mirrored version of the existing logic using point symmetry as their flight direction is rotated by 180° compared to a regular ship.
Besides, the current behavior of these ships in master can hardly be worsened. :-)